packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -29,7 +29,8 @@ import 'ticker_provider.dart';
 /// on the notification.
 ///
 /// Created automatically by [ScrollBehavior.buildViewportChrome] on platforms
-/// (e.g., Android) that commonly use this type of overscroll indication.
+/// (e.g., Android) that commonly use this type of overscroll indication with
+/// the [ThemeData.accentColor] color.
```

Comment:
ScrollBehavior.buildViewportChrome doesn't use the accentColor, it uses white.
MaterialApp overrides this to use the accent color (see _MaterialScrollBehavior).

Reply:
ditto